openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx

```diff
@@ -375,277 +585,250 @@ const AddAlertPage = () => {
+                                        className="w-4"
+                                        icon={Icons.DELETE}
+                                      />
+                                    }
+                                    type="text"
+                                    onClick={() => remove(name)}
+                                  />
                                 </div>
-                                <Button
-                                  data-testid={`remove-filter-rule-${name}`}
-                                  icon={
-                                    <SVGIcons
-                                      alt={t('label.delete')}
-                                      className="w-4"
-                                      icon={Icons.DELETE}
-                                    />
-                                  }
-                                  type="text"
-                                  onClick={() => remove(name)}
-                                />
                               </div>
-
-                              {fields.length - 1 !== key && <Divider />}
-                            </div>
-                          ))}
-                          <Form.Item>
-                            <Button
-                              block
-                              icon={<PlusOutlined />}
-                              type="dashed"
-                              onClick={() => add()}>
-                              {t('label.add-entity', {
-                                entity: t('label.destination'),
-                              })}
-                            </Button>
-                          </Form.Item>
-                        </>
-                      )}
-                    </Form.List>
-                  </Space>
-                </Col>
-                <Col className="footer" span={24}>
-                  <Button onClick={() => history.goBack()}>
-                    {t('label.cancel')}
-                  </Button>
-                  <Button htmlType="submit" type="primary">
-                    {t('label.save')}
-                  </Button>
-                </Col>
-              </Row>
+                            ))}
+                            <Form.ErrorList errors={errors} />
+                          </>
+                        )}
+                      </Form.List>
+                    </Space>
+                  </Col>
+                  <Col className="footer" span={24}>
+                    <Button onClick={() => history.goBack()}>
+                      {t('label.cancel')}
+                    </Button>
+                    <Button htmlType="submit" type="primary">
+                      {t('label.save')}
+                    </Button>
+                  </Col>
+                </Row>
+              </Form.Item>
             </Card>
           </Form>
         </Col>
+        <Col span={24} />
+        <Col span={24} />
```

Comment:
is it intentional to add 2 empty `<Col span={24} />`?

Reply:
Yes to avoid overlap of save button with Slack chat icon